controlplane/kubeadm/internal/proxy/dial.go

```diff
@@ -26,7 +26,9 @@ import (
 	"github.com/pkg/errors"
```

Comment:
nit: Could it be that PR title and description are slightly incorrect?

It seems to me that we are always using SPDY. The question is just if it's over webhooks or directly?